Link to libsupc++ instead of bringing in libstdc++ just because tomsocomp
is written in C++. It does not use STL so it needs not the whole standard
library.
Index: tvtime-1.0.2/src/Makefile.am
===================================================================
--- tvtime-1.0.2.orig/src/Makefile.am
+++ tvtime-1.0.2/src/Makefile.am
@@ -77,7 +77,7 @@ tvtime_CFLAGS = $(TTF_CFLAGS) $(PNG_CFLA
 	$(PLUGIN_CFLAGS) $(X11_CFLAGS) $(XML2_FLAG) \
 	$(FONT_CFLAGS) $(AM_CFLAGS)
 tvtime_LDFLAGS  = $(TTF_LIBS) $(ZLIB_LIBS) $(PNG_LIBS) \
-	$(X11_LIBS) $(XML2_LIBS) -lm -lstdc++
+	$(X11_LIBS) $(XML2_LIBS) -lm -lsupc++
 
 tvtime_command_SOURCES = utils.h utils.c tvtimeconf.h tvtimeconf.c \
 	tvtime-command.c
